_scripts/configs/cirrus.env.development

```diff
@@ -0,0 +1,11 @@
+CIRRUS_REMOTE_SETTING_URL=https://firefox.settings.services.allizom.org/v1/buckets/main/collections/nimbus-web-experiments/records
+CIRRUS_REMOTE_SETTING_PREVIEW_URL=https://firefox.settings.services.allizom.org/v1/buckets/main/collections/nimbus-web-preview/records
+CIRRUS_REMOTE_SETTING_REFRESH_RATE_IN_SECONDS=10
+CIRRUS_APP_ID=accounts.cirrus
+CIRRUS_APP_NAME=accounts_cirrus
+CIRRUS_CHANNEL=development
+CIRRUS_FML_PATH=/cirrus/feature_manifest/frontend-experiments.fml.yml
+CIRRUS_SENTRY_DSN=
```

Comment:
I see `SENTRY_DSN` is used in a bunch of places so I wonder if I could just use that, and if so, how can I test it out locally?

```suggestion
CIRRUS_SENTRY_DSN= # example value: https://12345.ingest.sentry.io/12345
```